### PR TITLE
Update asset state correctly even if locked

### PR DESCRIPTION
**Pull request by AdrienClairembault**
An uninstallation profile may edit an asset state:

image

However, if the field is locked nothing happens.

This is caused by de8f5911fa66bfbeae87ea7cbcfb851732998def which add the `is_dynamic` key to the input, which trigger the `cleanLockeds` function:

image

This change should be kept because it's needed to prevent fields from being locked as the `manageLocks` function check the same `is_dynamic` input:

image

TLDR: it's not possible to bypass existing locks (need `is_dynamic` = false) and to not lock the modified fields (need `is_dynamic` = true).

To fix this, I've manually removed the lock before the update and readded it after.
Maybe GLPI core should offer better options to deals with this.

### CODE REVIEW

**Comment by stonebuzz** [comment]
Maybe we could just check if ```LockField``` exist for ```states_id```?

If yes, just drop ```is_dynamic``` key from input 

Else let ```is_dynamic```.

**Comment by AdrienClairembault** [comment]
The update is just `is_dynamic` (which you added) and `states_id`:

image

Dropping `is_dynamic` if `states_id` exist = always dropping `states_id` = reverting the changes you made.
